Explorer/Assets/DCL/Character/CharacterMotion/Platforms/PlatformRaycast.cs

```diff
@@ -9,30 +10,31 @@ namespace DCL.CharacterMotion.Platforms
     public static class PlatformRaycast
     {
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static void Execute(CharacterPlatformComponent platformComponent, float radius, Transform transform, ICharacterControllerSettings settings)
+        public static void Execute(CharacterPlatformComponent platformComponent, float radius, Transform characterTransform, ICharacterControllerSettings settings)
         {
             float rayDistance = settings.PlatformRaycastLength;
             float halfDistance = (rayDistance * 0.5f) + radius;
 
+            Vector3 rayOrigin = characterTransform.position + (Vector3.up * halfDistance);
+
             var ray = new Ray
             {
-                origin = transform.position + (Vector3.up * halfDistance),
+                origin = rayOrigin,
                 direction = Vector3.down,
             };
 
-            bool hasHit = Physics.SphereCast(ray, radius, out RaycastHit hitInfo, rayDistance + radius, PhysicsLayers.CHARACTER_ONLY_MASK);
-
-            if (hasHit)
+            if (!DCLPhysics.SphereCast(ray, radius, out RaycastHit hitInfo, rayDistance + radius, PhysicsLayers.CHARACTER_ONLY_MASK, QueryTriggerInteraction.Ignore))
+                platformComponent.CurrentPlatform = null;
+            else if (platformComponent.CurrentPlatform != hitInfo.collider.transform)
             {
-                if (platformComponent.CurrentPlatform != hitInfo.collider.transform)
-                {
-                    platformComponent.CurrentPlatform = hitInfo.collider.transform;
-                    platformComponent.LastAvatarRelativePosition = platformComponent.CurrentPlatform.InverseTransformPoint(transform.position);
-                    platformComponent.LastAvatarRelativeRotation = platformComponent.CurrentPlatform.InverseTransformDirection(transform.forward);
-                }
+                platformComponent.PlatformCollider = hitInfo.collider;
+                platformComponent.CurrentPlatform = hitInfo.collider.transform;
+
+                platformComponent.LastPlatformPosition = hitInfo.collider.transform.position;
+                platformComponent.LastPlatformPosition = null;
+                platformComponent.LastAvatarRelativePosition = platformComponent.CurrentPlatform.InverseTransformPoint(characterTransform.position);
+                platformComponent.LastAvatarRelativeRotation = platformComponent.CurrentPlatform.InverseTransformDirection(characterTransform.forward);
```

Comment:
Is this intended? You are assigning the value for `platformComponent.LastPlatformPosition` twice and the second is `null`, maybe we could remove line 33?

Reply:
good catch 🎣